dvc/system.py

```diff
@@ -138,15 +138,16 @@ def getdirinfo(path):
         import ctypes
         from ctypes import c_void_p, c_wchar_p, Structure, WinError, POINTER
         from ctypes.wintypes import DWORD, HANDLE, BOOL
+        from win32file import (
+            FILE_FLAG_BACKUP_SEMANTICS,
+            FILE_FLAG_OPEN_REPARSE_POINT,
+            FILE_SHARE_READ,
+            OPEN_EXISTING,
+        )
 
         # NOTE: use this flag to open symlink itself and not the target
```

Comment:
Let's move it up to FILE_FLAG_OPEN_REPARSE_POINT

Reply:
or maybe just change to "NOTE: use FILE_FLAG_OPEN_REPARSE_POINT to open ..." so we don't lose context.